source/assets/styles/pages/_terms.scss

```diff
@@ -1,5 +1,5 @@
 .terms {
-  background-color: #f5f5f5;
+  background-color: #fff;
```

Comment:
:+1: Looks good. Alternately, in order to reduce code, we could remove this line completely (and the BG would be based on the standard body of the rest of it)